front/src/utils/date.ts

```diff
@@ -19,16 +18,19 @@ export function formatIsoDate(date: Date) {
 }
 
 export function dateTimeFormatting(date: Date, withoutTime: boolean = false) {
+  let locale;
   switch (i18n.language) {
     case 'fr':
-      dayjs.locale('fr');
+      locale = 'fr';
       break;
     default:
-      dayjs.locale('en-gb');
+      locale = 'en';
   }
-  const dateToUTC = dayjs(`${date}Z`); // The 'Z' is to ensure we have an UTC date
+  // Force interpreting the date in UTC
+  const dateToUTC = dayjs(date).utc(true);
```

Comment:
Maybe we should just drop the UTC conversion altogether here? (In practice browsers ignore the `Z` suffix, so dropping the UTC conversion would retain current behavior. I tried to keep the UTC conversion to retain the original intent of the old code, but maybe that's not a good idea.)

Reply:
we can discuss that if you want but it's seems good to me

Reply:
Alright, let's leave it as-is for now. Would be nice to audit all of the call-sites at some point to find out if any `Date` values really need this.